Formula/weboob.rb

```diff
@@ -2,11 +2,11 @@ class Weboob < Formula
   include Language::Python::Virtualenv
 
   desc "Web Outside of Browsers"
-  homepage "http://weboob.org/"
+  homepage "https://weboob.org/"
   url "https://symlink.me/attachments/download/356/weboob-1.3.tar.gz"
+  mirror "http://ftp.jaist.ac.jp/pub/Linux/Gentoo/distfiles/weboob-1.3.tar.gz"
```

Comment:
I'd suggest this secure mirror instead:
https://deb.debian.org/debian/pool/main/w/weboob/weboob_1.3.orig.tar.gz

Their new `head` URL is:
https://git.weboob.org/weboob/weboob.git

(I've sent the project an email a few days ago to repair the canonical download URL.)

Reply:
Mirror fixes when merging, head not readded because it's clearly not used and I don't have time to test it now.